src/hopen/syntax/silly_j.cljc

```diff
@@ -0,0 +1,51 @@
+(ns hopen.syntax.silly-j
+  (:require [hopen.renderer.xf :as rxf]
+            [instaparse.core :as insta]))
+
+(declare parse)
+
+;; TODO test case
+(defn render
```

Comment:
I would suggest to let the user do his own composition between the renderers and the parsers.

If this function is here just for testing, it makes sense to move it to the test namespace.